dbt_subprojects/daily_spellbook/models/near/ft_transfer_calls/near_ft_transfer_calls.sql

```diff
@@ -0,0 +1,76 @@
+{{
+  config(
+    schema = 'near_ft_transfer_calls',
+    alias = 'ft_transfer_calls',
+    materialized = 'table',
+    post_hook = '{{ expose_spells(\'["near"]\',
+                                    "project",
+                                    "ft_transfer_calls",
+                                    \'["danzipie"]\') }}'
+  )
+}}
+
+WITH
+    ft_transfer_calls AS (
+        SELECT
+            block_date,
+            block_height,
+            block_time,
+            block_hash,
+            chunk_hash,
+            shard_id,
+            tx_hash,
+            action_function_call_args_parsed,
+            receipt_id
+        FROM
+            {{ source('near', 'actions') }}
+        WHERE
+            action_function_call_call_method_name = 'ft_transfer_call'
```

Comment:
once you make incremental, we'll want to add the universal if is incremental logic filter. you can find plenty of examples across spellbook.